scala/BUILD

```diff
@@ -0,0 +1 @@
+java_library(name = "transitive_scalatest", exports = ["@scalatest//jar", "@scalactic//jar"])
```

Comment:
I think so, that's what actually creates a target that depends on scalatest and scalactic, no?